tests/migrations/test_operations.py

```diff
@@ -124,6 +125,14 @@ def set_up_test_model(
                     ("friend", models.ForeignKey("self", models.CASCADE))
                 ],
             ))
+        if o2o_model:
+            operations.append(migrations.CreateModel(
+                "Favorite",
```

Comment:
Maybe a name like PonyProfile would be easier to understand in the sense that it parallels extending User with a UserProfile. "Favorite" isn't so intuitive about how the model might relate to the others.